sphinx_gallery/gen_rst.py

```diff
@@ -737,23 +782,11 @@ def __call__(self):
 
 
 def _get_memory_base():
-    """Get the base amount of memory used by running a Python process."""
+    """Get the base amount of memory used by the current Python process."""
     # There might be a cleaner way to do this at some point
     from memory_profiler import memory_usage
 
-    if sys.platform in ("win32", "darwin"):
-        sleep, timeout = (1, 2)
-    else:
-        sleep, timeout = (0.5, 1)
-    proc = subprocess.Popen(
-        [sys.executable, "-c", f"import time, sys; time.sleep({sleep}); sys.exit(0)"],
-        close_fds=True,
-    )
-    memories = memory_usage(proc, interval=1e-3, timeout=timeout)
-    proc.communicate(timeout=timeout)
-    # On OSX sometimes the last entry can be None
-    memories = [mem for mem in memories if mem is not None] + [0.0]
-    memory_base = max(memories)
```

Comment:
Just for my info - why are we deleting? :grimacing:

Reply:
It never really worked robustly. On macOS it almost always errored to open and measure the subprocess